src/Dashboard/App_Start/AppModule.cs

```diff
@@ -35,46 +37,81 @@ public override void Load()
             }
 
             CloudBlobClient blobClient = account.CreateCloudBlobClient();
-            CloudQueueClient queueClient = account.CreateCloudQueueClient();
-
             Bind<CloudStorageAccount>().ToConstant(account);
             Bind<CloudBlobClient>().ToConstant(blobClient);
-            Bind<CloudQueueClient>().ToConstant(queueClient);
-
-            Bind<IHostVersionReader>().To<HostVersionReader>();
-            Bind<IDashboardVersionManager>().To<DashboardVersionManager>();
-            Bind<IFunctionInstanceLookup>().To<FunctionInstanceLookup>();
-            Bind<IFunctionInstanceLogger>().To<FunctionInstanceLogger>();
-            Bind<IHostIndexManager>().To<HostIndexManager>();
-            Bind<IFunctionIndexVersionManager>().To<FunctionIndexVersionManager>();
-            Bind<IFunctionIndexManager>().To<FunctionIndexManager>();
-            Bind<IFunctionLookup>().To<FunctionLookup>();
-            Bind<IFunctionIndexReader>().To<FunctionIndexReader>();
-            Bind<IHeartbeatValidityMonitor>().To<HeartbeatValidityMonitor>();
-            Bind<IHeartbeatMonitor>().To<HeartbeatMonitor>();
-            Bind<Cache>().ToConstant(HttpRuntime.Cache);
-            Bind<IFunctionStatisticsReader>().To<FunctionStatisticsReader>();
-            Bind<IFunctionStatisticsWriter>().To<FunctionStatisticsWriter>();
-            Bind<IRecentInvocationIndexReader>().To<RecentInvocationIndexReader>();
-            Bind<IRecentInvocationIndexWriter>().To<RecentInvocationIndexWriter>();
-            Bind<IRecentInvocationIndexByFunctionReader>().To<RecentInvocationIndexByFunctionReader>();
-            Bind<IRecentInvocationIndexByFunctionWriter>().To<RecentInvocationIndexByFunctionWriter>();
-            Bind<IRecentInvocationIndexByJobRunReader>().To<RecentInvocationIndexByJobRunReader>();
-            Bind<IRecentInvocationIndexByJobRunWriter>().To<RecentInvocationIndexByJobRunWriter>();
-            Bind<IRecentInvocationIndexByParentReader>().To<RecentInvocationIndexByParentReader>();
-            Bind<IRecentInvocationIndexByParentWriter>().To<RecentInvocationIndexByParentWriter>();
-            Bind<IHostMessageSender>().To<HostMessageSender>();
-            Bind<IPersistentQueueReader<PersistentQueueMessage>>().To<PersistentQueueReader<PersistentQueueMessage>>();
-            Bind<IFunctionQueuedLogger>().To<FunctionInstanceLogger>();
-            Bind<IHostIndexer>().To<HostIndexer>();
-            Bind<IFunctionIndexer>().To<FunctionIndexer>();
-            Bind<IIndexer>().To<UpgradeIndexer>();
-            Bind<IInvoker>().To<Invoker>();
-            Bind<IAbortRequestLogger>().To<AbortRequestLogger>();
-            Bind<IAborter>().To<Aborter>();
-
-            Bind<IIndexerLogWriter>().To<IndexerBlobLogWriter>();
-            Bind<IIndexerLogReader>().To<IndexerBlobLogReader>();
+
+            string tableLog = ConfigurationManager.AppSettings["AzureWebJobsLogTableName"];
```

Comment:
I'd prefer to add a bug to track this. 

---
In reply to: [56221610](https://github.com/Azure/azure-webjobs-sdk/pull/655#discussion_r56221610) [](ancestors = 56221610)